src/test/java/org/broadinstitute/hellbender/engine/FeatureManagerUnitTest.java

```diff
@@ -148,6 +151,48 @@ public void testHandleRequestForValidFeatureInputs() {
         Assert.assertEquals(bedFeatures.size(), 1, "Wrong number of Features returned from BED test Feature file");
     }
 
+    @Test
+    public void testHandleRequestForValidFeatureInputIterator() {
+        final ValidFeatureArgumentSource toolInstance = new ValidFeatureArgumentSource();
+
+        // Initialize two of the FeatureInput fields as they would be initialized by the argument-parsing
+        // system to simulate a run of the tool with two FeatureInputs.
+        toolInstance.variantContextFeatureInput = new FeatureInput<>(FEATURE_MANAGER_TEST_DIRECTORY + "feature_data_source_test.vcf");
+        toolInstance.bedListFeatureInput.add(new FeatureInput<>(FEATURE_MANAGER_TEST_DIRECTORY + "minimal_bed_file.bed"));
+
+        final FeatureManager manager = new FeatureManager(toolInstance);
+        final Iterator<VariantContext> vcIterator = manager.getFeatureIterator(toolInstance.variantContextFeatureInput);
+        final Iterator<BEDFeature> bedIterator = manager.getFeatureIterator(toolInstance.bedListFeatureInput.get(0));
+
+        final List<VariantContext> variants = StreamSupport.stream(Spliterators.spliteratorUnknownSize(vcIterator,0),false)
+                .collect(Collectors.toList());
+
+        final List<BEDFeature> beds = StreamSupport.stream(Spliterators.spliteratorUnknownSize(bedIterator,0),false)
+                .collect(Collectors.toList());
+
+        Assert.assertEquals(variants.size(), 26);
+        Assert.assertEquals(variants.get(4).getStart(),280);
+        Assert.assertEquals(beds.size(), 1);
+    }
+
+    @Test
+    public void testHandleRequestForValidFeatureInputIteratorWithoutIndex() {
+        final ValidFeatureArgumentSource toolInstance = new ValidFeatureArgumentSource();
+
+        // Initialize two of the FeatureInput fields as they would be initialized by the argument-parsing
+        // system to simulate a run of the tool with two FeatureInputs.
```

Comment:
Looks like a copy-paste error in this comment -- this test is only initializing one `FeatureInput`, not two.